libc/libc.go

```diff
@@ -1,16 +1,29 @@
 // Copyright The OpenTelemetry Authors
 // SPDX-License-Identifier: Apache-2.0
 
-package tpbase // import "go.opentelemetry.io/ebpf-profiler/tpbase"
+package libc // import "go.opentelemetry.io/ebpf-profiler/libc"
 
 import (
 	"debug/elf"
 	"fmt"
 	"regexp"
 
+	"go.opentelemetry.io/ebpf-profiler/internal/log"
+
 	"go.opentelemetry.io/ebpf-profiler/libpf/pfelf"
 )
 
+type LibcInfo struct {
+	DTVInfo *DTVInfo
+	TSDInfo *TSDInfo
+}
+
+type DTVInfo struct {
+	Offset     int64  // Offset of DTV from FS base (or from thread pointer)
+	EntryWidth uint32 // Width of each DTV entry in bytes
+	Indirect   uint8  // 0 if DTV is at FS+offset, 1 if at [FS+0]+offset
```

Comment:
nit: typically we follow godoc documentation pattern: comment is before the field, and the comment sentence starts with the field/func/whatever name.

Reply:
Fixed and moved to C, renamed and change sizes to match TSDInfo while i was at it.